hca/staging/cli/select_command.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+from .config_store import ConfigStore
+from .staging_area_urn import StagingAreaURN
+
+
+class SelectCommand:
+
+    def __init__(self, args):
+        self.config = ConfigStore()
+        if re.search(':', args.urn_or_alias):  # URN
+            self._save_and_select_area_by_urn(args.urn_or_alias)
+        else:  # alias
+            self._select_area_by_alias(args.urn_or_alias)
+        self.config.save()
+
+    def _save_and_select_area_by_urn(self, urn_string):
+        urn = StagingAreaURN(urn_string)
+        self.config.add_area(urn)
+        self._select_area(urn.uuid)
+
+    def _select_area_by_alias(self, alias):
+        matching_areas = [uuid for uuid in self.config.areas() if re.match(alias, uuid)]
+        if len(matching_areas) == 0:
+            print("Sorry I don't recognize area \"%s\"" % (alias,))
```

Comment:
If you're going to do the `(xxx,)` thing for a single element tuple, you should be consistent about doing it everywhere.